test/e2e/cli_test.go

```diff
@@ -19,6 +20,8 @@ type CLISuite struct {
 
 func (s *CLISuite) BeforeTest(suiteName, testName string) {
 	s.E2ESuite.BeforeTest(suiteName, testName)
+	_ = os.Setenv("ARGO_SERVER", "localhost:2746")
```

Comment:
I refactored this into a new file named `env.go`, so you can probably delete it?